### PR TITLE
feat: bump chartmuseum image

### PR DESCRIPTION
**Pull request by palexster**
**What problem does this PR solve?**:

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: services/chartmuseum/3.10.1/chartmuseum-3.10.0.tgz
Comment (mikolajb) [inline services/chartmuseum/3.10.1/chartmuseum-3.10.0.tgz:1]: `3.10.1.tgz`?

Comment (palexster) [reply services/chartmuseum/3.10.1/chartmuseum-3.10.0.tgz:1]: Has it been already released?

Comment (mikolajb) [reply services/chartmuseum/3.10.1/chartmuseum-3.10.0.tgz:1]: Yes! We are waiting for `3.10.2` 😇 

https://github.com/chartmuseum/charts/releases/tag/chartmuseum-3.10.1

**Review by mikolajb — APPROVED** [review]

**Comment by kaiwalyajoshi** [comment]
@palexster @mikolajb 

@cbuto mentioned that it may take a bit more time to sign and get the charts released.
The charts are in a seperate repo [here](https://github.com/chartmuseum/charts/tree/main) and hasn't had any updates for 7 months.

Can we use the existing chart `v3.10.0` but with the newer `v0.16.1` image?

**Pull request merged**